### ISSUE TITLE
In-App Questionnaire - Questionnaire

### ISSUE THREAD

**Issue by ALuckette** [influxdata/ui#6092]
## ACs
- The questionnaire should match the [figma](https://www.figma.com/file/V2R5zW3VbPZevhRBj3wmlo/Sign-up-Questionnaire?node-id=6%3A1374) design: icons to the left of each question. 3 questions total. 
- Display all questions at once. Do NOT auto-select any options for the user. 
- All questions should be required as indicated by a red asterisk to the right of the question. 
- Users should only be able to select one option for the "What are you looking to do on InfluxDB" question. 
- Users should be able to select multiple options for the "What kind of data do you plan to bring into InfluxDB" question. 
- The options for "What is your preferred language?" should be in a type-ahead dropdown (use existing typeahead dropdown component)
- The "Continue" button should be disabled until the user submits an answer for all 3 questions.

**Comment by eatondustin1** [comment]
TLDR: Suggesting we consider _not_ disabling the "Continue" button.

@ALuckette I was reviewing the PR, and I wanted to make a suggestion that's really more for design. There are pros and cons to either approach, but when we block a user action preemptively by disabling a UI element like a form submit button, we lose the ability to give the user feedback on _why_ the user can't take the action. If instead, we don't disabled the button, we can provide reactive feedback, like form validation errors, when the user attempts the action.

Well placed and unambiguous error messages on their own are usually a good enough indicator to a user on their own, but the exception is when a user first encounters a form and they haven't attempted to fill it out yet. Because it's so easy to just skim information, a user might see that the form is a survey and assume it isn't that important, and look directly for the "Continue" button. If it's disabled, they would then have to go back to the form and see the asterisks to know that there are required fields; but if instead, they were able to click on the "Continue" button, they would immediately see feedback from the validation errors shown on the form.
